src/static/js/keys.js

```diff
@@ -32,7 +32,7 @@ function fetchAndDisplayUserKeys() {
             `;
 
             const row = `
-            <tr onclick="showKeyDetails('${hostFriendlyName}', '${publicKey}', ${itemDescription})">
+            <tr onclick="showKeyDetails('${hostFriendlyName}', '${publicKey}', '${itemDescription}')">
```

Comment:
It seems a small fix of HTML element content.

Reply:
Yap. We just lost `'` 🥲